Documentation Standard Updates (#33)
fivetran/dbt_asana: README.md

</p>

# Asana dbt Package ([Docs](https://fivetran.github.io/dbt_asana/))
## What does this dbt package do?

- Produces modeled tables that leverage Asana data from [Fivetran's connector](https://fivetran.com/docs/applications/asana) in the format described by [this ERD](https://fivetran.com/docs/applications/asana#schemainformation) and builds off of the output of our [Asana source package](https://github.com/fivetran/dbt_asana_source).

- Enhances the task, users, projects, teams, and tags tables. Each of these tables is enriched with metrics that reflect the volume and breadth of current work and also the velocity of completed work.
- Provides a daily metrics table, which lays out a timeline of task creations and completions to show the overall pace of deliverables.
- Generates a comprehensive data dictionary of your source and modeled Asana data through the [dbt docs site](https://fivetran.github.io/dbt_asana/).
- These tables are designed to work simultaneously with our [Asana source package](https://github.com/fivetran/dbt_asana_source).

> The following table provides a detailed list of all models materialized within this package by default.

Author: ```suggestion

<!--section=“asana_transformation_model"-->
The following table provides a detailed list of all tables materialized within this package by default.
```

> TIP: See more details about these models in the package's [dbt docs site](https://fivetran.github.io/dbt_asana/#!/overview?g_v=1&g_e=seeds).

> | **Model**                | **Description**                                                                                                                                |

Author: ```suggestion
| **Todel**                | **Description**                                                                                                                                |
```

| [asana_project](https://fivetran.github.io/dbt_asana/#!/model/model.asana.asana__project)          | Each record represents an Asana project, enriched with metrics about their completed tasks, open tasks, and the users involved in the project. Also includes data about the project's most recently completed task and next due tasks. |
| [asana__team](https://fivetran.github.io/dbt_asana/#!/model/model.asana.asana__team)             | Each record represents an Asana team, enriched with data about their completed tasks, open tasks, their projects, and the users involved with the team. |
| [asana__tag](https://fivetran.github.io/dbt_asana/#!/model/model.asana.asana__tag)              | Each record represents an Asana tag, enriched with metrics about open and completed tasks associated with the tag. |
| [asana__daily_metrics](https://fivetran.github.io/dbt_asana/#!/model/model.asana.asana__daily_metrics)    | Each record represents a single day, enriched with metrics about tasks opened at created that day. |

> 

Author: ```suggestion
<!--section-end-->
```
